src/mtginator/game.py

```diff
@@ -92,28 +106,39 @@ def main(self):
         self.make_play(plays)
 
     def land_drop(self):
-        lands = [c for c in self.hand if c.is_land]
-        pick = random.choice(lands)
-        # Note: should implment some color optimization
-        pick.play()
-        print("Played {} as Land for turn [ hand size: {} ]".format(pick, len(self.hand)))
+        lands = [c for c in self.hand if c.is_land()]
+        if len(lands) > 0:
+            pick = random.choice(lands)
+            # Note: should implment some color optimization
+            pick.play(None) #lands don't need context to play
```

Comment:
oh here you have a null context, probably this should assume a board state/context, even if it's irrelvant for land drops